client/src/components/inspirationPage/inspiration/inspCard/inspCard.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './inspCard.css';
+
+export const InspCardDetails = ({ description,isVisible }) => (
```

Comment:
I would call replace Insp with Inspo, makes more semantic sense.

Reply:
Is it a big deal?
Shall I change it?

Reply:
Remember this when you're refactoring!